templates/communityteams_list.html

```diff
@@ -0,0 +1,28 @@
+{% extends "layout.html" %}
+{% block title %}{{ this.title }}{% endblock %}
+{% block body %}
+  <p>This is a list of all CC Community Teams members, and their roles and privileges.</p>
```

Comment:
I think we should call this "CC Community Team", it's one big team. That needs to be updated other places too though.